src/etl/arbo/index.ts

```diff
@@ -0,0 +1,86 @@
+import Airtable from "airtable";
+import { pipe } from "fp-ts/lib/function.js";
+import { AirtableEstimateFields, AirtableSourceFields } from "./types.js";
+import { cleanFieldNamesAndRemoveUnusedFieldsStep } from "./steps/clean-field-names-and-remove-unused-fields-step.js";
+import { cleanSingleElementArrayFieldsStep } from "./steps/clean-single-element-array-fields-step.js";
+import { parseDatesStep } from "./steps/parse-dates-step.js";
+import { removeEstimatesWithLowSampleSizeStep } from "./steps/remove-estimates-with-low-sample-size-step.js";
+import { mergeEstimatesAndSourcesStep } from "./steps/merge-estimates-and-sources-step.js";
+import { latLngGenerationStep } from "./steps/lat-lng-generation-step.js";
+import { removeRecordsThatAreFlaggedToNotSaveStep } from "./steps/remove-records-that-are-flagged-to-not-save-step.js";
+import { jitterPinLatLngStep } from "./steps/jitter-pin-lat-lng-step.js";
+import { transformIntoFormatForDatabaseStep } from "./steps/transform-into-format-for-database-step.js";
+import { assertMandatoryFieldsArePresentStep } from "./steps/assert-mandatory-fields-are-present-step.js";
+import { transformNotReportedValuesToUndefinedStep } from "./steps/transform-not-reported-values-to-undefined-step.js";
+import { addCountryAndRegionInformationStep } from "./steps/add-country-and-region-information-step.js";
+import { asyncEtlStep, etlStep, getEnvironmentVariableOrThrow, getMongoClient, writeDataToMongoEtlStep } from "../helpers.js";
+
+const runEtlMain = async () => {
```

Comment:
No actual functionality change, just using some helper functions to make this a bit less verbose